src/launchWizard/launchWizard.ts

```diff
@@ -188,10 +188,9 @@ async function createWizard(ctx: vscode.ExtensionContext) {
           panel.dispose()
```

Comment:
Comment about above code that isn't part of this change set.

This is really a suggestion for the future. 

The most common thing I want to do with a config is clone one, then update it. 

You currently offer create a new one, update an existing one. But I don't see copy one. 

Having gone through the tedium of configuring one config for a test I want to use while debugging, for the most part I want to clone it and just change the test data file to be used.

Reply:
Added this a new issue > https://github.com/apache/daffodil-vscode/issues/670